src/libs/ajax/WorkspaceDataService.ts

```diff
@@ -70,6 +85,21 @@ export const WorkspaceData = (signal) => ({
     );
     return res.json();
   },
+  getCapabilities: async (root: string): Promise<Capabilities> => {
+    return fetchWDS(root)('capabilities/v1', _.mergeAll([authOpts(), { signal, method: 'GET' }]))
+      .then(async (response) => {
+        const json = await response.json();
+        const capabilities: Capabilities = _.mapValues((value) => value === true, json);
+        return capabilities;
+      })
+      .catch((error) => {
+        if (error instanceof Response && error.status === 404) {
+          console.log("WDS doesn't support capabilities endpoint"); // eslint-disable-line no-console
```

Comment:
what's the benefit of having a console.log here at all? It won't really help end users, and engineers could just look for the 404 in dev tools.

Reply:
Mainly I wanted proof that this safety/fallback check on 404 was working in local dev mode.  I'll drop it.

Reply:
Dropped in e9b2ec419e08f77b234995e2061f7f46c52f1ad3